.env.example

```diff
@@ -48,4 +48,9 @@ DOMAIN=localhost
 FEEDBACK_FORWARD_EMAIL=
 
 BACKEND_URL=http://localhost:4200
-NEXT_PUBLIC_BACKEND_URL=http://localhost:4200
\ No newline at end of file
+NEXT_PUBLIC_BACKEND_URL=http://localhost:4200
+
+
+# Enter time in secs
+THROTTLE_TTL=
```

Comment:
Add these inside `apps/api/src/env/env.ts`

Reply:
Okay will do

Reply:
But these are Config variables, how they would be accessed if I put them in a ts file?